scalapact-shared/src/main/scala/com/itv/scalapact/shared/PactsForVerificationResponse.scala

```diff
@@ -34,14 +35,40 @@ import com.itv.scalapact.shared.Pact.Links
 
 
 final case class PactsForVerificationResponse(_embedded: EmbeddedPactsForVerification, _links: Links) {
-  def pacts: List[EmbeddedPactForVerification] = _embedded.pacts
+  def pacts: List[PactForVerification] = _embedded.pacts
 }
 
-final case class EmbeddedPactsForVerification(pacts: List[EmbeddedPactForVerification]) extends AnyVal
+final case class EmbeddedPactsForVerification(pacts: List[PactForVerification]) extends AnyVal
 
-final case class EmbeddedPactForVerification(verificationProperties: VerificationProperties, _links: Links) {
+final case class PactForVerification(verificationProperties: VerificationProperties, _links: Links) {
   def href: Option[String] = _links.get("self").map(_.href)
-  def notices: List[Map[String, String]] = verificationProperties.notices
 }
 
-final case class VerificationProperties(notices: List[Map[String, String]]) extends AnyVal
+sealed trait VerificationProperties {
+  type A <: Notice
+  def pending: Boolean
+  def notices: List[A]
+}
+
+object VerificationProperties {
+  sealed trait VerificationPropertiesAux[A0 <: Notice] extends VerificationProperties { type A = A0 }
+
+  final case class SimpleVerificationProperties(notices: List[SimpleNotice])
+    extends VerificationPropertiesAux[SimpleNotice] {
+    val pending = false
+  }
+
+  final case class PendingStateVerificationProperties(pending: Boolean, notices: List[PendingStateNotice]) extends VerificationPropertiesAux[PendingStateNotice]
+
+}
+
+sealed trait Notice {
+  def text: String
+}
+
+object Notice {
```

Comment:
The logic should really only care about "before verification" and "after verification" notices. I want to be able to put anything in here, and have it display automatically without any client changes. `PendingStateNotice` seems to be putting business logic into the code that I don't think it should know about.